Source/WebCore/css/StyleRule.cpp

```diff
@@ -99,11 +99,10 @@ template<typename Visitor> constexpr decltype(auto) StyleRuleBase::visitDerived(
         return std::invoke(std::forward<Visitor>(visitor), downcast<StyleRuleLayer>(*this));
     case StyleRuleType::Container:
         return std::invoke(std::forward<Visitor>(visitor), downcast<StyleRuleContainer>(*this));
-    case StyleRuleType::Margin:
-        RELEASE_ASSERT_NOT_REACHED();
-    case StyleRuleType::Unknown:
-        RELEASE_ASSERT_NOT_REACHED();
+    default:
+        break;
     }
+    RELEASE_ASSERT_NOT_REACHED();
```

Comment:
Wouldn't it be better to _not_ have a `default:` case? With this modification when enum elements get added, the compiler will not let us know that there are cases not being handled. I prefer to list the cases that are supposed to be unreachable explicitly, all together:

```suggestion
    case StyleRuleType::Margin:
    case StyleRuleType::Unknown:
    case StyleTuleType:: ... :
        RELEASE_ASSERT_NOT_REACHED();
    }
```

Reply:
Sure.